pkg/apis/build/v1beta1/build_types.go

```diff
@@ -191,6 +192,10 @@ type BuildSpec struct {
 	// +patchMergeKey=Key
 	// +patchStrategy=merge
 	Tolerations []corev1.Toleration `json:"tolerations,omitempty" patchStrategy:"merge" patchMergeKey:"Key"`
+
+	// SchedulerName specifies the scheduler to be used to dispatch the Pod
+	// +optional
+	SchedulerName string `json:"schedulerName,omitempty"`
```

Comment:
We have a general API convention that optional fields are pointers. Will require chances at other places.

```suggestion
	SchedulerName *string `json:"schedulerName,omitempty"`
```